ops/ansible/group_vars/all/vars_file.yml

```diff
@@ -0,0 +1,3 @@
+---
+remote_user: "ubuntu"
```

Comment:
Yeah, that's an idea. Like create a PostMaster user?

What other options are there? Can we just use built-in users instead?